build/build-images.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+# Set image registry as a variable
+IMAGE_REGISTRY=${IMAGE_REGISTRY:-quay.io/sustainable_computing_io}
+
+# Detect whether podman or docker exists
+if command -v podman &> /dev/null
+then
+    DOCKER_CMD="podman"
+else
+    DOCKER_CMD="docker"
+fi
+
+# set ARCH variable for base image build, if it is x86_64, then use amd64
+ARCH=$(uname -m)
+if [ "$ARCH" == "x86_64" ]
+then
+    ARCH="amd64"
+fi
+
+# Set image tag prefixes
+BASE_TAG_PREFIX="ubi-9-"
+BUILDER_TAG_PREFIX="ubi-9-"
+KEPLER_TAG_PREFIX="ubi-9-"
+
+# Set image tag suffixes
+BCC_TAG_SUFFIX="bcc-0.26"
+LIBBPF_TAG_SUFFIX="libbpf-1.2.0"
+
+# Set image tags
+KEPLER_BCC_TAG="latest"
+KEPLER_LIBBPF_TAG="latest-libbpf"
+
+# Parse command line options
+BUILD_BASE=false
+BUILD_BUILDER=false
+BUILD_KEPLER=false
+PUSH_ONLY=false
+
+function show_usage() {
+    echo "Usage: build-images.sh [OPTIONS]"
+    echo "Builds Kepler images."
+    echo ""
+    echo "Options:"
+    echo "  -b, --base      Build base image"
+    echo "  -r, --builder   Build builder image"
+    echo "  -k, --kepler    Build Kepler image"
+    echo "  -p, --push      Push images only"
```

Comment:
Do we really need to support Push-only scenario?
If the `build-images.sh` is only used in CI, the built-out images normally should be pushed into registry.